demos/shadow/shadow_demo_main/CMakeLists.txt

```diff
@@ -53,12 +53,6 @@ if(ROOT_CA_CERT_PATH)
             ROOT_CA_CERT_PATH="${ROOT_CA_CERT_PATH}"
     )
 endif()
-if(BROKER_ENDPOINT)
-    target_compile_definitions(
-        ${DEMO_NAME} PRIVATE
-            BROKER_ENDPOINT="${BROKER_ENDPOINT}"
-    )
-endif()
```

Comment:
Should this be renamed to `AWS_IOT_ENDPOINT` instead?

Reply:
It is already supported through the `check_aws_credentials` call here: https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/master/demos/shadow/shadow_demo_main/CMakeLists.txt#L18